views/practice/spacedpractice.html

```diff
@@ -0,0 +1,140 @@
+{{extend 'admin/instructors.html'}}
```

Comment:
I don't see a controller corresponding to this. Did I miss it?

Reply:
No. I had actually created two controllers for spaced repetition and self-regulated learning. Then you decided to remove the self-regulated and I removed the controller, but forgot to remove this file.

Reply:
It looks like this file is still in the repo...